lambda-functions/jira-bot-filter-task-runner/condition-checker.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+const config = require('./config');
+
+lastHumanActivityDate = issue => {
+  const humanHistories = issue.changelog.histories.filter(history => {
```

Comment:
sorry, i mis-read the line. you are removing the BOTS that are commented. at first, i assumed you were removing the author of the issue, but i now see it is the 'author' of the history itself and you are ensuring it is not a bot. nice.